satellite/satellitedb/storagenodeaccounting.go

```diff
@@ -200,21 +200,49 @@ func (db *StoragenodeAccounting) QueryPaymentInfo(ctx context.Context, start tim
 func (db *StoragenodeAccounting) QueryStorageNodeUsage(ctx context.Context, nodeID storj.NodeID, start time.Time, end time.Time) (_ []accounting.StorageNodeUsage, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	query := `SELECT at_rest_total, start_time, 
-		LAG(at_rest_total) OVER win AS prev_at_rest, 
-		LAG(start_time) OVER win AS prev_start_time
-		FROM accounting_rollups
-		WHERE id IN (
-			SELECT MAX(id)
-			FROM accounting_rollups
-			WHERE node_id = ?
-			AND ? <= start_time AND start_time <= ?
-			GROUP BY start_time
-			ORDER BY start_time ASC
-		)
-		WINDOW win AS (ORDER BY start_time)`
+	lastRollup, err := db.db.Find_AccountingTimestamps_Value_By_Name(ctx, dbx.AccountingTimestamps_Name(accounting.LastRollup))
```

Comment:
Is this query needed with the last changes?

Before the `lastRollup` value was used in the function, now it's only used for making this function to return earlier. If that's wanted, then fine, otherwise, we can remove it.

Reply:
Yes, it used to check whether we have rollup run at least once and `LastRollup` timestamp is not nil. Then we querying this value acquiring the same reading lock as the main query. This ensures that we are not blocking dB for other reads, and allows `rollup` and other services that writes to dB to acquire pending lock if needed so we are not blocking their work

Reply:
I think that I'm not at the level to understand that lock acquiring.
If you know that's needed it's fine to me unless that anybody else has something to say about.

Reply:
Well I just try to say that new query can avoid locking 2 db calls into TX which is blocking dB for read and writes

Reply:
so it can be removed, isn't it?

Reply:
We don’t need to execute second query if LastRollup is null

Reply:
How often `latRollup` is null?

Reply:
Before very first rollup